### PR TITLE
update eth gas station api url

### PR DESCRIPTION
**Pull request by cds95**
## Description

The ETH Gas Station API has been returning errors due to the EA making calls to a bad endpoint.  This PR updates the endpoint to the correct one.

## Changes

- Update endpoint called within the EthGasStation EA

## Steps to Test

1. Run EA and send request below
```
{
  "jobRunID": "1",
  "data": {
      "endpoint":"gasprice",
      "speed":"fast"
    }
}
```
2. Verify successful response

## Quality Assurance

- [x] Ran `yarn changeset` if adapter source code was changed
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `<ADAPTER_PACKAGE>/schemas/env.json` and `<ADAPTER_PACKAGE>/README.md`
- [x] If a new adapter was made, or an existing one was modified so that its environment variables have changed, update the relevant `infra-k8s` configuration file.
- [x] The branch naming follows git flow (`feature/x`, `chore/x`, `release/x`, `hotfix/x`, `fix/x`) or is created from Clubhouse/Shortcut
- [x] This is related to a maximum of one Clubhouse/Shortcut story or GitHub issue
- [x] Types are safe (avoid TypeScript/TSLint features like any and disable, instead use more specific types)

### CODE REVIEW

File: packages/sources/ethgasstation/src/config.ts [hunk 1]
```diff
@@ -4,7 +4,7 @@ import { Config } from '@chainlink/types'
 export const NAME = 'ETHGASSTATION'
 
 export const DEFAULT_ENDPOINT = 'gasprice'
-export const DEFAULT_BASE_URL = 'https://data-api.defipulse.com'
+export const DEFAULT_BASE_URL = 'https://ethgasstation.info/'
```
[the remark this answers is not in the document]
Comment (austinborn) [reply packages/sources/ethgasstation/src/config.ts:7]: `API_ENDPOINT` would be the place to add other endpoints to override `DEFAULT_BASE_URL`, right?

**Review by austinborn — APPROVED** [review]

**Review by austinborn — CHANGES_REQUESTED** [review]
Need to fix integration tests

File: packages/sources/ethgasstation/src/config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (cds95) [reply packages/sources/ethgasstation/src/config.ts:7]: the problem is that the path is also different and I had to change that as well.  The new path won't work with the old URL.

**Comment by austinborn** [comment]
Integration tests are not using the mocks for some reason

**Review by austinborn — APPROVED** [review]

File: packages/sources/ethgasstation/src/config.ts (hunk 1 above)
[the remark this answers is not in the document]
Comment (austinborn) [reply packages/sources/ethgasstation/src/config.ts:7]: If the team likes the idea, how about we add a ticket for it?

**Pull request merged**